packages/editor/src/components/Panel.js

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import { useEffect } from 'react'
+import { jsx, useColorMode } from 'theme-ui'
+
+import { Heading } from './Typography'
+import Colors from './Panel/Colors'
+import Modes from './Panel/Modes'
+import Typography from './Panel/Typography'
+import CopyThemeToClipboard from './CopyThemeToClipboard'
+
+const Panel = ({ state, setColorMode, setTheme }) => {
+  const [panelColorMode, setPanelColorMode] = useColorMode()
+
+  useEffect(() => {
+    if (window.chrome.devtools.panels.themeName === 'dark') {
```

Comment:
I think any devtools related code should be decoupled from the editor UI components and go in the devtools source code instead

Reply:
I've moved this into the `chrome` package 👍